infra/modules/providers/azure/app-monitoring/README.md

```diff
@@ -0,0 +1,92 @@
+
+#### More on Terraform Resources
+
+Terraform resources used to define the `app-monitoring` module include the following:
+
+- `azurerm_monitor_metric_alert` - https://www.terraform.io/docs/providers/azurerm/r/monitor_metric_alert.html
+
+- `azurerm_monitor_action_group` - https://www.terraform.io/docs/providers/azurerm/r/monitor_action_group.html
+
+## Characteristics
+
+A Terraform module that deploys the "Azure Monitor" service in order to provide Cobalt Templates with the following characteristics:
+
+- Ability to deploy Azure Monitor in the same resource group as the App Service Plan.
+
+- Ability to deploy Azure Monitor with a single set of configurable `metric alert criteria` targeting an Azure resource.
+
+- Ability to deploy Azure Monitor with a single set of configurable `metric alert criteria` tied to a single `action group`.
+
+### Default Behavior
+
+The `action group` variables are configured to send e-mail notifications when `metric alert criteria` crosses configured threshold values. Values used to define `metric alert criteria` are provided by a template. If a template does not provide an e-mail by satisfying the `action_group_email_receiver` variable, this module will not be present in the Terraform plan generated for deployment.
+
+- Required Variables:
+  - resource_group_name               
+  - resource_ids                      
+  - action_group_name                 
+  - action_group_email_receiver       
+  - metric_alert_criteria_namespace   
+  - metric_alert_name                 
+  - metric_alert_criteria_name        
+  - metric_alert_criteria_aggregation 
+  - metric_alert_criteria_operator    
+  - metric_alert_criteria_threshold
+
+### Module Usage
+
+- Service Plan Module : infra/modules/providers/azure/service-plan
+- App Monitoring Module :  infra/modules/providers/azure/app-monitoring
+
+```
+variable "resource_group_name" {
+  value = "test-rg"
+}
+
+variable "service_plan_name" {
+  value = "test-svcplan"
+}
+
+module "service_plan" {
+  resource_group_name     = "${var.resource_group_name}"
+  resource_group_location = "${var.resource_group_location}"
+  service_plan_name       = "${var.service_plan_name}"
+}
+
+module "app_monitoring" {
+  source                            = "../../modules/providers/azure/app-monitoring"
+  resource_group_name               = "${module.service_plan.resource_group_name}"
+  resource_ids                      = ["${module.service_plan.app_service_plan_id}"]
+  action_group_name                 = "${var.action_group_name}"
+  action_group_email_receiver       = "${var.action_group_email_receiver}"
+  metric_alert_criteria_namespace   = "${var.metric_alert_criteria_namespace}"
+  metric_alert_name                 = "${var.metric_alert_name}"
```

Comment:
Can you provide examples of what these values (lines 73 -80) might be + a link to a document on Azure docs that describes their meaning?

Reply:
A link to the Azure doc's is now provided in the sub-sections titled "Configuring **metric alert criteria**" and "Configuring **action group**"